src/renderer/pages/replays/replay_file_stats/player_info.tsx

```diff
@@ -85,7 +87,16 @@ export const PlayerInfo = ({
               font-weight: 500;
             `}
           >
-            {connectCode}
+            <A
+              css={css`
+                &:hover {
+                  opacity: 0.7;
+                }
+              `}
+              href={slippiProfileUrl}
+            >
+              <div>{connectCode}</div>
```

Comment:
```suggestion
              {connectCode}
```
can you remove the nested div here? we shouldn't need it.